src/backend/gtk3/message_dialog.rs

```diff
@@ -86,6 +88,17 @@ impl GtkMessageDialog {
     }
 }
 
+/// Sets the child labels of a widget selectable
+unsafe fn set_child_labels_selectable(dialog: *mut gtk_sys::GtkDialog) {
+    let area = gtk_sys::gtk_message_dialog_get_message_area(dialog as _);
+    let mut children = gtk_sys::gtk_container_get_children(area as _);
+    while !children.is_null() {
+        let child = (*children).data;
+        gtk_sys::gtk_label_set_selectable(child as _, 1);
```

Comment:
I'm not checking here if the child is a label.
Don't know if it's necessary.
Don't know what the equivalent of `GTK_IS_LABEL` is in `gtk_sys`.

Reply:
As far as I can tell `GTK_IS_LABEL` compiles down to casting the pointer to `GTypeInstance`, then extracting the type out of it like so: `instance.g_class.g_type`, after that it can be compared with `gtk_sys::gtk_label_get_type()`.

Proceed with caution, I'm not well versed in that either, just checked what C implementation does.